client/src/components/ResultsList.js

```diff
@@ -1,24 +1,19 @@
 import React, { useRef } from "react";
-import SelectedStakeholderDisplay from "./ResultsSelectedStakeholder";
+import StakeholderDetails from "./StakeholderDetails";
 import PropTypes from "prop-types";
-import moment from "moment";
-import mapMarker from "../images/mapMarker";
 import { Grid } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
-import pantryIcon from "../images/pantryIcon";
-import mealIcon from "../images/mealIcon";
-import splitPantryMealIcon from "../images/splitPantryMealIcon";
-import {
-  MEAL_PROGRAM_CATEGORY_ID,
-  FOOD_PANTRY_CATEGORY_ID,
-} from "../constants/stakeholder";
-import { ORGANIZATION_COLORS, CLOSED_COLOR } from "../constants/map";
```

Comment:
Moved StakeholderPreview and Details logic to their own components or a shared helper function.

Reply:
Good. We need to do more re-factorign into components where we can.